### PR TITLE
(enhancement)docs: add docs section for litmus demo

### PR DESCRIPTION
**Pull request by ksatchit**
- Adds litmus-demo into documentation

### CODE REVIEW

**Review by uditgaurav — APPROVED** [review]

File: docs/litmus-demo.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+ --
```
Comment (uditgaurav) [inline docs/litmus-demo.md]: `---`

**Review by uditgaurav — CHANGES_REQUESTED** [review]

File: docs/litmus-demo.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    ```
+    watch kubectl get pods --all-namespaces
+    ```
+
+    Once all pods come into Running state we can access the sock-shop application through web-ui which will help us to 
+    visualize the impact of chaos on the application and whether the application persists after chaos injections. 
+
+  - Get the port of frontend deployment
+
+    ```kubectl get deploy front-end -n sock-shop -o jsonpath='{.spec.template.spec.containers[?(@.name == "front-end")].ports[0].containerPort}'
```
Comment (uditgaurav) [inline docs/litmus-demo.md]: kubectl should start from the next line I guess. This command is not visible.

**Comment by ksatchit** [comment]
Updated. Ptal @uditgaurav !

File: docs/litmus-demo.md (hunk 2 above)
Comment (ksatchit) [reply docs/litmus-demo.md]: Fixed! Thanks.

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**